storage/src/main/java/tech/pegasys/teku/storage/server/rocksdb/RocksDbDatabase.java

```diff
@@ -108,6 +118,82 @@ public StorageUpdateResult update(final StorageUpdate event) {
     return doUpdate(event);
   }
 
+  @Override
+  public StorageUpdateResult addEth1Deposit(final DepositWithIndex depositWithIndex) {
```

Comment:
No need to return a `StorageUpdateResult` - nothing uses it anyway.

Reply:
it seemed like a standard we might need, glad to clean that up.